### PR TITLE
Remove rxlifecycle interop

### PR DESCRIPTION
**Pull request by ZacSweers**
We can re-add if there's an rxlifecycle 4 with rxjava3 support

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -62,8 +62,6 @@ if (System.getenv("ANDROID_HOME") != null) {
 include ':autodispose'
 include ':autodispose-coroutines-interop'
 include ':autodispose-lifecycle'
-//include ':autodispose-rxlifecycle'
```
Comment (ShaishavGandhi) [inline settings.gradle]: Remove?

Comment (ZacSweers) [reply settings.gradle]: They are removed locally... ? 🤔

Comment (ZacSweers) [reply settings.gradle]: Yeah they are removed o_O

Comment (ShaishavGandhi) [reply settings.gradle]: Nvm i misread 🤦‍♂

**Pull request merged**